generators/client/files-angular.js

```diff
@@ -472,7 +477,8 @@ const files = {
                 'spec/app/account/password-reset/init/_password-reset-init.component.spec.ts',
                 'spec/app/account/password-reset/finish/_password-reset-finish.component.spec.ts',
                 'spec/app/account/register/_register.component.spec.ts',
-                'spec/app/account/settings/_settings.component.spec.ts'
+                'spec/app/account/settings/_settings.component.spec.ts',
```

Comment:
add comment `//login componenet tests` above to separate login related stuff from account stuff